### ISSUE TITLE
"Error" in file t81_558_class04_training.ipynb

### ISSUE THREAD

**Issue by Brotherlui** [jeffheaton/t81_558_deep_learning#19]
Hi, in the code cell for "Training with a Validation Set and Early Stopping" the dataset is split into training and validation set but the model is fitted with x,y instead of x_train, x_test
```
# Split into train/test
x_train, x_test, y_train, y_test = train_test_split(    
    x, y, test_size=0.25, random_state=42)
...
model.fit(x, y, validation_data=(x_test, y_test), callbacks=[monitor], verbose=2, epochs=1000)
```

**Comment by jeffheaton** [comment]
Thank you!
